plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseConfig.java

```diff
@@ -22,17 +22,31 @@
 {
     // TODO (https://github.com/trinodb/trino/issues/7102) reconsider default behavior
     private boolean mapStringAsVarchar;
+    private boolean replaceNullToDefault;
```

Comment:
`settings` table in ClickHouse provides `input_format_null_as_default` value. No need to add config properties.

Reply:
@zhengxingmao as pointed out, would you be able to add the required changes to read from system table instead of adding another config?

Reply:
Clickhouse allows null values to be inserted into non null declared columns, which by default converts null to the default value of the corresponding data type. This PR maintains the same behavior as Clickhouse when Trino writes null values for Clickhouse connector.So we need a config the control this behavior.

Reply:
@zhengxingmao I think the request is to avoid adding a new external config. Instead identify the value of newly defined property(`replaceNullToDefault`) through `input_format_null_as_default` property present in `settings` table present under `system` database.